### PR TITLE
Declare expenses

### PR DESCRIPTION
**Pull request by topheryun**
8 JUnit tests for AccountServiceImplTest.java

### CODE REVIEW

File: bin/.gitignore [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+/nbdist/
+/.nb-gradle/
+build/
+!**/src/main/**/build/
+!**/src/test/**/build/
+
+### VS Code ###
+.vscode/
+tim-gattie-firebase-service-account.json
+firebase-client-config
```
Comment (RevatureGentry) [inline bin/.gitignore]: 👍

**Review by RevatureGentry — CHANGES_REQUESTED** [review]
Let's not include our bytecode in commits.

**Comment by topheryun** [comment]
Tim, I removed the bin files and added bin/ to .gitignore

File: src/main/java/com/revature/personalfinance/repo/IAccountRepo.java [hunk 2]
```diff
@@ -9,8 +9,9 @@
 
     List<Account> getAccountsByUserId(int userId);
 
-    //Account updateAccount(Account account);
+//    Account updateAccount(Account account);
```
Comment (GattieTime) [inline src/main/java/com/revature/personalfinance/repo/IAccountRepo.java]: Why the change why the commented code?

**Review by GattieTime — CHANGES_REQUESTED** [review]

File: src/main/java/com/revature/personalfinance/repo/IAccountRepo.java (hunk 2 above)
Comment (topheryun) [reply src/main/java/com/revature/personalfinance/repo/IAccountRepo.java]: It was something we were intended to use, but was not needed in the end. Not sure why the file got committed. I tried a bunch of things to remove it from the PR, but I just ended up reverting the changes.

**Review by RevatureGentry — APPROVED** [review]

**Review by GattieTime — APPROVED** [review]
